Marlin/src/module/temperature.cpp

```diff
@@ -1718,10 +1833,10 @@ void Temperature::init() {
   #endif
 
   // Thermistor activation by MCU pin
-  #if PIN_EXISTS(TEMP_0_TR_ENABLE_PIN)
+  #if PIN_EXISTS(TEMP_0_TR_ENABLE)
     OUT_WRITE(TEMP_0_TR_ENABLE_PIN, ENABLED(HEATER_0_USES_MAX6675));
```

Comment:
I think this shows that the max code in marlin is completely broken...

Reply:
@rhapsodyv I searched for the use of TEMP_0_TR_ENABLE_PIN and it is only used by the LERDGE board

Reply:
The issues is not with `TEMP_0_TR_ENABLE_PIN`, is with the usage of `PIN_EXISTS`. You fixed it. We should not use `_PIN` in the `PIN_EXISTS` call. So, the old code (before the fix) will have detect that `TEMP_0_TR_ENABLE_PIN` exists... and will never call that OUT_WRITE.

It's that I told before. The max code in marlin is outdated and with a lot of issues.